test/expected.txt

```diff
@@ -124,6 +138,12 @@
 || 4 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy]
+|| 	1. Adam Brown Phone: 111111 Email: adam@gmail.com Address: 111, alpha street Tags: 
```

Comment:
Since your feature involves listing the address book on every command, it would help to modify the tests to show no user -> one use transition and the relevant outputs. (for sake of completeness)